### DISCUSSION TITLE
Janssen UI

### DISCUSSION THREAD

**Discussion by foanthoanGH** [JanssenProject/jans#5154] [Q&A]
Does Janssen has UI Capabilty or it just CLi?

**Comment by mzico** [comment]
@foanthoan : Janssen has TUI inside. But it has UI named "Flex" which require license. https://gluu.org/flex/

**Reply by ossdhaval** [reply]
Ref: [TIU docs](https://docs.jans.io/v1.0.13/admin/config-guide/jans-tui/)

**Comment by nynymike** [comment]
The TUI is the open source tool for interactive configuration. If you want a web UI, you can try [Gluu Flex](https://docs.gluu.org), which offers a free 30 day trial license if you just want to try it.
